.github/workflows/release_windows_pytorch_wheels.yml

```diff
@@ -2,6 +2,19 @@ name: Release Windows PyTorch Wheels
 
 on:
   workflow_dispatch:
+    inputs:
+      release_type:
+        description: The type of release to build ("nightly", or "dev")
+        type: string
+        default: "dev"
+      s3_subdir:
+        description: S3 subdirectory, not including the GPU-family
+        type: string
+        default: "v2"
+      cloudfront_url:
+        description: CloudFront URL pointing to Python index
+        type: string
+        default: "d25kgig7rdsyks.cloudfront.net/v2"
 
   # TODO(scotttodd): add schedule once working
   # schedule:
```

Comment:
FYI, I dropped running on schedule from the Linux workflow and instead trigger this workflow from the release portable Linux packages workflow.